scripts/check-branch-name.js

```diff
@@ -0,0 +1,8 @@
+// TODO: migrate to Imba, add colors and spinner
```

Comment:
Huh.. we need to make that works https://github.com/shapkarin/awesome-imba/blob/husky_protects_with_imba/scripts/check-branch-name.imba I'll try later

Reply:
I compiled `Imba` before, nut sure whats wrong